osu.Game/Tests/Visual/OsuManualInputManagerTestScene.cs

```diff
@@ -46,21 +46,23 @@ protected OsuManualInputManagerTestScene()
         {
             var mainContent = content = new Container { RelativeSizeAxes = Axes.Both };
 
+            var inputContainer = new Container { RelativeSizeAxes = Axes.Both };
+
             if (DisplayCursorForManualInput)
             {
                 var cursorDisplay = new GlobalCursorDisplay { RelativeSizeAxes = Axes.Both };
 
-                cursorDisplay.Add(new OsuTooltipContainer(cursorDisplay.MenuCursor)
+                cursorDisplay.Add(content = new OsuTooltipContainer(cursorDisplay.MenuCursor)
                 {
                     RelativeSizeAxes = Axes.Both,
-                    Child = mainContent
                 });
 
-                mainContent = cursorDisplay;
+                inputContainer.Add(cursorDisplay);
+                mainContent = inputContainer;
             }
 
             if (CreateNestedActionContainer)
-                mainContent = new GlobalActionContainer(null).WithChild(mainContent);
+                inputContainer.Add(new GlobalActionContainer(null));
```

Comment:
I can't understand what this code is doing anymore. Probably best to restructure into a single hierarchy if possible.

Reply:
Have simplified it. Not sure why I made another container.